src/data/device_registry.ts

```diff
@@ -16,8 +16,8 @@ export interface DeviceRegistryEntry {
 }
 
 export interface DeviceRegistryEntryMutableParams {
-  area_id?: string;
-  name_by_user?: string;
+  area_id?: string | null;
```

Comment:
```suggestion
  area_id?: string | undefined;
```

Reply:
It needs to allow null to unset it. Undefined means don't change it.

Reply:
Correct, `null` is a valid value. `undefined` doesn't get converted to JSON.